tools/GraphClust/GSPAN/structure_to_gspan.xml

```diff
@@ -0,0 +1,111 @@
+<tool id="gspan" name="structure_to_gspan" version="0.1">
+    <requirements>
+        <requirement type="package" version="0.2.0">graphclust-wrappers</requirement>
+    </requirements>
+    <stdio>
+        <exit_code range="1:" />
+    </stdio>
+    <command>
+    <![CDATA[
+        mkdir ./temp/ ./GSPAN_Outputs/ &&
+        structure_to_gspan.pl
+            --input-file '$dataFile'
+            --input-format 'rnafold'
+            $abstr
+            $stack
+            $seq_graph_t
+            -group $group
+            -tmp ./temp
+            -o ./GSPAN_Outputs/
+    ]]>
+    </command>
+    <inputs>
+        <param type="data" name="dataFile" format="data" label="Sequence and Structure input file in the prediction tool format" />
```

Comment:
format="data" does not exist. It needs to be fasta or something else. "txt" is also fine but very loosly defined.